protocol-designer/src/localization/en/create_new_protocol.json

```
@@ -1,19 +1,41 @@
 {
+  "add_custom_tips": "Add custom pipette tips",
   "add_fixtures": "Add your fixtures",
   "add_gripper": "Add a gripper",
   "add_modules": "Add your modules",
-  "add_pip": "Add a pipette",
+  "add_pip": "Add a pipette and tips",
+  "author_org": "Author/Organization",
   "basics": "Let’s start with the basics",
-  "need_gripper": "Does your protocol need a Flex Gripper?",
+  "description": "Description",
+  "edit": "Edit",
+  "fixtures_added": "Fixtures added",
+  "modules_added": "Modules added",
+  "name": "Name",
+  "need_gripper": "Do you want to move labware automatically with the gripper?",
+  "fixtures_replace": "Fixtures replace standard deck slots and let you add functionality to your Flex.",
+  "pip_gen": "Pipette generation",
+  "pip_tips": "Pipette tips",
   "pip_type": "Pipette type",
   "pip_vol": "Pipette volume",
-  "name": "Name",
-  "description": "Description",
-  "author_org": "Author/Organization",
+  "pip": "{{mount}} pipette",
+  "quantity": "Quantity",
   "questions": "We’re going to ask a few questions to help you get started building your protocol.",
-  "robot_type": "Which robot would you like to use?",
+  "remove": "Remove",
+  "robot_pips": "Robot pipettes",
+  "robot_type": "What kind of robot do you have?",
+  "show_all_tips": "Show all tips",
+  "show_default_tips": "Show default tips",
+  "stagingArea_cutoutA3": "Staging area A3",
+  "stagingArea_cutoutB3": "Staging area B3",
+  "stagingArea_cutoutC3": "Staging area C3",
+  "stagingArea_cutoutD3": "Staging area D3",
+  "swap": "Swap pipettes",
   "tell_us": "Tell us about your protocol",
+  "trashBin": "Trash bin",
+  "vol_label": "{{volume}} uL",
+  "wasteChute": "Waste chute",
   "which_fixtures": "Which fixtures will you be using?",
-  "which_mods": "Which modules will you be using?",
-  "which_pip": "Tell us what pipette and tips you want to use."
+  "which_mods": "Select modules to use in your protocol.",
+  "which_pip": "Pick your first pipette. If you need a second pipette, you can add it next.",
```

nit
Because someone might think of python

```suggestion
  "which_pipette": "Pick your first pipette. If you need a second pipette, you can add it next.",
```